repository/t.json

```
@@ -544,6 +544,19 @@
 				}
 			]
 		},
+		{
+			"name": "Telescope",
+			"details": "https://github.com/Mister7F/sublime-telescope",
+			"releases": [
+				{
+					"sublime_text": ">=4198",
+					"tags": true
+				}
+			],
+			"labels": [
+				"code navigation"
+			]
```

```suggestion
			"labels": ["code navigation"],
			"releases": [
				{
					"sublime_text": ">=4198",
					"tags": true
				}
			]
```

Just to follow conventional formatting here. Otherwise looks good 👍🏻